frontend/packages/kubevirt-plugin/src/components/modals/menu-actions-modals/DeleteVMTCustomizationModal.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
```

Comment:
because `deleteModal` uses resource `kind` to show messages and do logic. But in my case Im deleting VM but show messages about VM template so `kinds` do not match. Since its a quite specific use case, i opted for creating a new modal instead of altering existing one.